server/resources/translations/common.es_us.json

```diff
@@ -8,6 +8,9 @@
   "buttonLowBatteryInitial": "La batería del Button {{ buttonDisplayName }} está baja y necesita recargarse. Para ver un video que muestra cómo recargar la batería, vaya a https://www.youtube.com/watch?v=h7_5ff-_r5A",
   "buttonLowBatteryReminder": "La batería del Button {{ buttonDisplayName }} sigue baja y necesita recargarse.\n\nPara ver un video que muestra cómo recargar la batería, vaya a https://www.youtube.com/watch?v=h7_5ff-_r5A",
   "buttonLowBatteryNoLonger": "La batería del Button {{ buttonDisplayName }} ya no está baja. Gracias.",
+  "buttonStatusChangeDisconnected": "Los siguientes botones se han desconectado: {{ buttonDisplayNames }}.",
+  "buttonStatusChangeReconnected": "Los siguientes botones se han vuelto a conectar: {{ buttonDisplayNames }}.",
+  "buttonStatusChangeStart": "Ha habido cambios de conexión para los botones en {{ clientDisplayName }}.",
```

Comment:
Same here about replacing `botones` with `Buttons` (capital B) and about bugging Oona for translations. I'm not sure who she got the Spanish ones from. Thanks!